src/vector/index.js

```diff
@@ -250,16 +251,20 @@ async function loadApp() {
 
     if (!preventRedirect) {
         if (/iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream) {
-            if (confirm(_t("Riot is not supported on mobile web. Install the app?"))) {
+            window.location = "https://status.im/join-riot.html";
```

Comment:
Won't that cause problems for selfhosted Riots?

Reply:
thanks for noticing this.
Yes, please don't hijack people

Reply:
heh, looks like this crept in whilst my back was turned. of course it’s not meant to be there; will turn it into a config options. thanks!

Reply:
@ara4n Note a few lines below this is the same line again ;)